app/Services/GameService.php

```diff
@@ -8,25 +8,19 @@ class GameService {
 
     function getGamesById (int $id)
     {
-        $games = $this->getGames();
+        $game = Game::findOrFail($id);
+        return $game;
 
-        foreach ($games as $game) {
-            if ($game->id === $id) {
-                return $game;
-            }
-        }
-        throw new NotFoundHttpException();
     }
 
     // todo: add optional params for orderBy, direction, and limit
     function getGames()
     {
-        return
-            [
-                Game::make(['id' => 1, 'title' => 'Monopoly', 'rating' => 1]),
-                Game::make(['id' => 2, 'title' => 'Risk', 'rating' => 2]),
-                Game::make([ 'id' => 3, 'title' => 'DND', 'rating' => 5]),
-            ];
+        $allGames = Game::all();
+        $sortedAllGames = $allGames->sortBy('title');
+
+        return $sortedAllGames;
+
     }
 
     function searchGamesByTitle (string $searchTitle)
```

Comment:
you need to update this method as well, as i mentioned in the issue description

Reply:
I believe this was resolved as of 11/7

Reply:
we can resolve it once you push your changes, right now this method is unchanged in your branch.

Reply:
Okay I think most recent push has resolved this!